src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -2040,4 +2040,16 @@ Widening and user defined conversions are not supported with generic types.</val
   <data name="PreferIsEmptyOverAnyMessage" xml:space="preserve">
     <value>Prefer an 'IsEmpty' check rather than using 'Any()', both for clarity and for performance</value>
   </data>
+  <data name="DoNotIgnoreReturnValueMessage" xml:space="preserve">
+    <value>Do not ignore the return value of '{0}'</value>
+  </data>
+  <data name="DoNotIgnoreReturnValueDescription" xml:space="preserve">
+    <value>The return value from '{0}' is being ignored, but the method is annotated to indicate its return value should not be ignored.</value>
```

Comment:
Descriptions shouldn't have place holders. Placeholders are only substituted for messages.